elodie/plugins/plugins.py

```diff
@@ -0,0 +1,52 @@
+"""
+Plugin object.
+
+.. moduleauthor:: Jaisen Mathai <jaisen@jmathai.com>
+"""
+from __future__ import print_function
+from builtins import object
+
+from sys import exc_info
+from importlib import import_module
+
+from elodie.config import load_config
+from elodie import log
+
+
+class Plugins(object):
+    """A class to execute plugin actions."""
+
+    def __init__(self):
+        self.plugins = []
+        self.classes = {}
+
+    def load(self):
+        """Load plugins from config file.
+        """
+        config = load_config()
+        if 'Plugins' in config and 'plugins' in config['Plugins']:
+            config_plugins = config['Plugins']['plugins'].split(',')
```

Comment:
I feel like if you put this logic of loading and parsing the config file inside the https://github.com/jmathai/elodie/blob/ccd5ada7b2c7fd608221ab0cd4b5b2edf6eddba4/elodie/config.py#L10 function itself it would make the tests easier to write but also it would make it less error-prone to change the config file layout, if ever needed.

You could even have a unique `load_plugin_config()` function which would return directly the list of plugins. If the config file doesn't have the required path it would return an empty list.

Reply:
You're right. Thanks for the suggestion. I'll move the logic before merging.

Reply:
Moved code to `config.py`. There's still some logic in the plugins class to load modules which I kept out of the config class.